site/content/docs/5.1/components/toasts.md

```diff
@@ -49,7 +49,7 @@ Previously, our scripts dynamically added the `.hide` class to completely hide a
 
 Click the button below to show a toast (positioned with our utilities in the lower right corner) that has been hidden by default.
 
-<div class="position-fixed bottom-0 end-0 p-3" style="z-index: 11">
+<div class="toast-container position-fixed bottom-0 end-0 p-3" style="z-index: 11">
```

Comment:
Should be safe to drop the inline styles for `z-index` in examples, shouldn't it?